app/src/main/java/com/duckduckgo/app/browser/shortcut/ShortcutBuilder.kt

```diff
@@ -39,7 +39,7 @@ class ShortcutBuilder @Inject constructor() {
         intent.putExtra(SHORTCUT_EXTRA_ARG, true)
 
         val icon = when {
-            homeShortcut.url == USE_OUR_APP_SHORTCUT_URL -> IconCompat.createWithResource(context, R.drawable.ic_fb_favicon)
+            useOurAppDetector.isUseOurAppUrl(homeShortcut.url) -> IconCompat.createWithResource(context, R.drawable.ic_fb_favicon)
```

Comment:
Checks for the domain rather the full URL for better comparison against the control.